wazimap_ng/profile/admin/forms/profile_indicator_form.py

```diff
@@ -5,9 +5,26 @@
 from django_json_widget.widgets import JSONEditorWidget
 
 from ... import models
+from wazimap_ng.datasets.models import Indicator
+from wazimap_ng.config.common import (
+    DENOMINATOR_CHOICES, PERMISSION_TYPES, PI_CONTENT_TYPE
+)
 
 
 class ProfileIndicatorAdminForm(forms.ModelForm):
+
+    content_type = forms.ChoiceField(choices=PI_CONTENT_TYPE, required=False)
+    content_indicator = forms.ModelChoiceField(
+        queryset=Indicator.objects.filter(dataset__content_type="qualitative"),
+        widget=VariableFilterWidget, required=False
+    )
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args,**kwargs)
```

Comment:
How do I create a new qualitative indicator?

Reply:
Removed this part of code